client/src/util.ts

```diff
@@ -27,7 +25,7 @@ export async function getDenoCommand(): Promise<string> {
     // if sent a relative path, iterate over workspace folders to try and resolve.
     const list = [];
     for (const workspace of workspaceFolders) {
-      const dir = path.resolve(workspace.uri.path, command);
+      const dir = path.resolve(workspace.uri.fsPath, command);
```

Comment:
Bug on windows. Should be `fsPath` in order to remove the leading slash (ex. shoudl be `v:/folder` and not `/v:/folder`).